pkgs/top-level/all-packages.nix

```diff
@@ -4650,7 +4650,6 @@ with pkgs;
   qastools = libsForQt5.callPackage ../tools/audio/qastools { };
 
   qesteidutil = libsForQt5.callPackage ../tools/security/qesteidutil { } ;
-  qdigidoc = libsForQt5.callPackage ../tools/security/qdigidoc { } ;
```

Comment:
I think it's better to mark it as broken instead of removing.

Reply:
Why should we spend time on evaluation on derivations that have not seen any updates since 2016? Also it removes a large patch from the channel.